tests/start_proxy/start_proxy_test.py

```diff
@@ -138,6 +138,24 @@ def test_gen_proxy_config(self):
               '--listener_port', '8080', '--ssl_client_cert_path',
               '/etc/endpoint/ssl', '--disable_tracing'
               ]),
+            # ssl_client_root_certs_file specified
+            (['-R=managed','--listener_port=8080',  '--disable_tracing',
+              '--ssl_client_root_certs_file=/etc/endpoints/ssl/ca-certificates.crt' ],
+             ['bin/configmanager', '--logtostderr',
+              '--backend_address', 'http://127.0.0.1:8082',
+              '--rollout_strategy', 'managed', '--v', '0',
+              '--listener_port', '8080', '--root_certs_path',
+              '/etc/endpoints/ssl/ca-certificates.crt', '--disable_tracing'
+              ]), 
+            # legacy enable_grpc_backend_ssl specified
+            (['-R=managed','--listener_port=8080',  '--disable_tracing',
```

Comment:
add unit test of non default grpc_backend_ssl_root_certs_file

Reply:
Ack. Not necessary.